pre_commit/color.py

```diff
@@ -48,7 +48,7 @@ def use_color(setting):
     if setting not in COLOR_CHOICES:
         raise InvalidColorSetting(setting)
 
-    if 'NO_COLOR' in os.environ:
+    if 'NO_COLOR' in os.environ and os.environ['NO_COLOR']:
```

Comment:
`if os.environ.get('NO_COLOR')` would be ~slightly simpler

Reply:
Ok, I will change that.